modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types are consistent across all ClusterVersion and ClusterOperator conditions. Individual components may also have additional condition types with component-specific language.
+
+The condition type, as well as additional informatin about your operator can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o yaml
+----
+
+
+* Available: 
+The condition with type `Available` indicates that the operators are functional and available in the cluster. If the status is `False`, then the status indicates that at least one part of the operand is non-functional and the condition requires an administrator to intervene.
```

Comment:
nit: "If the status is `False`, then the status indicates", feels stuttery on "status".  Maybe:

> If the status is `False`, then the condition indicates...
> If the status is `False`, that indicates...

or some such?